content/en/blog/2025/opentelemetry-logging-and-you.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Opentelemetry Logging and You
+linkTitle: OTel Logging # Mandatory, make sure that your short title.
+date: 2025-04-16 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Austin Parker](https://github.com/austinlparker) (OpenTelemetry)
+issue: the issue ID for this blog post # TODO: See https://opentelemetry.io/docs/contributing/blog/ for details
+sig: Logging # TODO: add the name of the SIG that sponsors this blog post
+---
+
+If you’ve been following OpenTelemetry for a while, you’ve probably heard a lot
+about logs. Logging bridges, Logging APIs, Events, you name it, we’ve talked
+about it. This blog post is intended to be a discussion of the rationale and
+current design direction of logging in OpenTelemetry.
+
+## Definitions
+
+Let’s get started with a basic definition of how OpenTelemetry thinks about
+logs. Broadly, logs are any telemetry that is emitted through a LoggerProvider,
```

Comment:
```suggestion
logs. Broadly, log records are any telemetry that is emitted through a `Logger`,
```

Reply:
I thought it was still LoggerProvider?

Reply:
Maybe what you have written is actually OK. LoggerProvider returns a Logger. Logger is used to emit log records (and events). Feel free to resolve my comment if you find the current wording correct.